h2o-algos/src/main/java/hex/ensemble/StackedEnsemble.java

```diff
@@ -176,7 +176,14 @@ private Frame prepareTrainingLevelOneFrame(StackedEnsembleModel.StackedEnsembleP
         }
       }
 
-      return prepareLevelOneFrame(levelOneKey, baseModels.toArray(new Model[0]), baseModelPredictions.toArray(new Frame[0]), _model._parms.train());
+      Frame levelOneFrame = prepareLevelOneFrame(levelOneKey, baseModels.toArray(new Model[0]), baseModelPredictions.toArray(new Frame[0]), _model._parms.train());
+      if (_parms._keep_levelone_frame) {
+        levelOneFrame.write_lock(_job);
+        levelOneFrame = levelOneFrame.deepCopy(levelOneFrame._key.toString());
+        levelOneFrame.unlock(_job);
+        DKV.put(levelOneFrame);
```

Comment:
Perhaps a little nicer is to use 

   levelOneFrame.update(_job)
   levelOneFrame.unlock(_job)

Reply:
yep! changed.